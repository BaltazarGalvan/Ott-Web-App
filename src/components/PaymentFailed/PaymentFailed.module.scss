@use '../../styles/variables';
@use '../../styles/theme';

.title {
  font-family: theme.$body-font-family;
  font-weight: 700;
  font-size: 24px;
}

.message {
  font-family: theme.$body-font-family;
  font-size: 16px;
}
