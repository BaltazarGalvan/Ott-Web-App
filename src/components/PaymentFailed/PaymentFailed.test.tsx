import React from 'react';
import { render } from '@testing-library/react';

import PaymentFailed from './PaymentFailed';

describe('<PaymentFailed>', () => {
  test('renders and matches snapshot', () => {
    const { container } = render(<PaymentFailed type="error" onCloseButtonClick={jest.fn()} message="Error message" />);

    expect(container).toMatchSnapshot();
  });
});
