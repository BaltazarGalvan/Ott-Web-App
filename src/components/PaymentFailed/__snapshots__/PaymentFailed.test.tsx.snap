// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaymentFailed> renders and matches snapshot 1`] = `
<div>
  <div>
    <h2
      class="title"
    >
      checkout.payment_error
    </h2>
    <p
      class="message"
    />
    <div>
      <button
        class="button primary fullWidth large"
      >
        <span>
          checkout.close
        </span>
      </button>
    </div>
  </div>
</div>
`;
